test/commons/text/unicode.js

```diff
@@ -71,7 +71,7 @@ describe('text.hasUnicode', function() {
 		});
 
 		it('returns true for a string with characters in supplementary private use area A', function() {
-			var actual = axe.commons.text.hasUnicode('\uDB80\uDC19', {
+			var actual = axe.commons.text.hasUnicode('\uDB80\uDFFE', {
```

Comment:
Updated these tests so that they pass if the regex is:

```js
// The change, as of this PR
/[\uDB80-\uDBBF][\uDC00-\uDFFF]/g
```

but fail if it's

```js
/[\uDB80-\uDBBF][\uDC00-\uDFFD]/g
```